docs/en/datasets/explorer/dashboard.md

```
@@ -35,46 +35,45 @@ pip install ultralytics[explorer]
+<img width="1709" alt="Explorer Ask AI results for images with 5 persons" src="https://cdn.jsdelivr.net/gh/ultralytics/assets@main/docs/explorer-dashboard-screenshot-4.avif">
 </p>
 
-Note: This works using [Large Language Models](https://www.ultralytics.com/glossary/large-language-model-llm) under the hood so the results are probabilistic and might get things wrong sometimes
+Note: This feature uses [Large Language Models](https://www.ultralytics.com/glossary/large-language-model-llm), so results are probabilistic and may be inaccurate.
 
 ## Run SQL queries on your CV datasets
 
-You can run SQL queries on your dataset to filter it. It also works if you only provide the WHERE clause. Example SQL query would show only the images that have at least one 1 person and 1 dog in them:
+You can run SQL queries on your dataset to filter it. It also works if you only provide the WHERE clause. For example, the following WHERE clause returns images that contain at least one person and one dog:
 
 ```sql
```

markdownlint-cli2 (0.20.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)